src/snyk/snykOss/services/vulnerabilityCount/parsers/packageJsonParser.ts

```diff
@@ -52,15 +68,28 @@ export class PackageJsonParser extends ModuleParser {
     }
 
     const depLineIndex = lines.indexOf(depStartLine) + 1;
-    const depLines = lines.slice(depLineIndex, depLineIndex + Object.keys(dependencies).length);
+    let dependenciesLength = Object.keys(dependencies).length;
+    const emptyLineTabRegex = /(\t)$/;
```

Comment:
It included "empty lines" in dependencies. Then it compared the dependencies length to the lines length which was different in case there was an empty line.